examples/complete/main.tf

```diff
@@ -31,6 +60,16 @@ module "kms_root_key" {
   source          = "../.."
   kms_instance_id = ibm_resource_instance.key_protect_instance.guid
   key_name        = "${var.prefix}-root-key"
+  kmip = [
+    {
+      name = "${var.prefix}-kmip-adapter"
+      certificates = [
+        {
+          certificate = split("\n\n", data.ibm_sm_public_certificate.kmip_cert.certificate)[0]
```

Comment:
why do we have to split and add the newlines? What does the output look like here?

Reply:
This was due to the certificate output having the concatenated intermediate certificate on it, not sure if it is intended or a bug. We've switched to using the `private cert` now which doesn't have the same concatenation